### PR TITLE
Update go-tarantool in comparison table (code coverage, doc, static analysis)

### PR DESCRIPTION
**Pull request by ligurio**
Follows up https://github.com/tarantool/go-tarantool/issues/142
Follows up https://github.com/tarantool/go-tarantool/issues/123
and https://github.com/tarantool/go-tarantool/pull/149

### CODE REVIEW

File: doc/book/connectors/__go.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Last update: January 2022
         -   12
 
     *   -   Static analysis
-        -   No
+        -   Yes
```
Comment (Totktonada) [inline doc/book/connectors/__go.rst]: We mention a tool for other connectors and it looks meaningful to do that here too.

**Review by Totktonada — APPROVED** [review]
LGTM aside of one tiny comment.

File: doc/book/connectors/__go.rst (hunk 1 above)
Comment (ligurio) [reply doc/book/connectors/__go.rst]: Ok.

```diff
--- a/doc/book/connectors/__go.rst
+++ b/doc/book/connectors/__go.rst
@@ -52,7 +52,7 @@ Last update: January 2022
         -   12
 
     *   -   Static analysis
-        -   Yes
+        -   golangci-lint, luacheck
         -   golint
         -   golangci-lint
 
```

File: doc/book/connectors/__go.rst [hunk 2]
```diff
@@ -37,7 +37,7 @@ Last update: January 2022
         -   2021
 
     *   -   Documentation
-        -   README with examples, API described in code comments
+        -   README with example, up-to-date GoDoc
```
Comment (NickVolynkin) [inline doc/book/connectors/__go.rst]: with a single example?

Comment (NickVolynkin) [reply doc/book/connectors/__go.rst]:
```suggestion
        -   README with examples and up-to-date GoDoc
```

Comment (ligurio) [reply doc/book/connectors/__go.rst]: Updated, thanks

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**